presidio-anonymizer/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 89
```

Comment:
Will have to be aligned with black, @omri374 are we going with a custom length different from PEP8 from a special reason?

Reply:
Black uses 88 which is not aligned with PEP8 but makes more sense with modern monitors. This should also be 88 and not 89. Agree.

Reply:
@omri374  it should also be changed in the analyzer to 88. The value is currently similar in both.